projects/js-packages/publicize-components/src/hooks/use-sync-post-data-to-store/test/index.test.js

```diff
@@ -13,7 +13,7 @@ import {
 const connections = connectionsList.map( connection => ( { ...connection, enabled: true } ) );
 
 const post = {
-	jetpack_publicize_connections: [ connections[ 0 ] ],
+	jetpack_publicize_connections: [ connections ],
```

Comment:
cc @manzoorwanijk because you wrote this testcase:
I think this change won't alter the testing power of this test, I also think that it should never be the case that prevConnections is not the same length as freshConnections

Reply:
Shouldn't it then be this instead?

```suggestion
	jetpack_publicize_connections: connections,
```